Sources/Sentry/include/SentryArrayFilter.h

```diff
@@ -0,0 +1,13 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryArrayFilter : NSObject
```

Comment:
H: I'm probably missing something but do we need this public API?
Doesn't seem to be used anywhere let alone publicly

Reply:
I committed this by accident. Good catch 👏